validator/testcases/javascript/instanceproperties.py

```diff
@@ -225,6 +252,8 @@ def set_contentScript(value, traverser):
                         "is dangerous and error-prone. Please use a separate "
                         "JavaScript file, along with the "
                         "`contentScriptFile` property instead.",
+            signing_help="Please do not use the `contentScript` property "
+                         "in any add-ons submitted for automated signing.",
```

Comment:
Yeah, sorry I wasn't clear. They are blanket banned in unlisted, non-sideload add-ons. They are very likely to cause rejection in other add-ons unless they're simple, static strings.